tests/unittests/QuantizationTest.cpp

```diff
@@ -839,7 +839,8 @@ class MockQuantBackend : public Backend {
   }
   bool isOpSupported(Kinded::Kind opKind, ElemKind elementTy) const override {
     if (opKind == Kinded::Kind::SoftMaxNodeKind ||
-        opKind == Kinded::Kind::LocalResponseNormalizationNodeKind) {
+        opKind == Kinded::Kind::LocalResponseNormalizationNodeKind ||
+        opKind == Kinded::Kind::SaveNodeKind) {
```

Comment:
Why was this added? Is it just to make sure that the check in `canConvert()` overrides the preferences of the backend's `isOpSupported()`? If so perhaps a check should be added in the `quantizeSoftmaxAndLRN` test that checks that the Save isn't quantized?

Reply:
All backends need to support SaveNode, I wanted to make sure that quantization procedure handles Save node well.

Reply:
```
If so perhaps a check should be added in the quantizeSoftmaxAndLRN test that checks that the Save isn't quantized?
```

Additional checks will be useful.

Reply:
>All backends need to support SaveNode, I wanted to make sure that quantization procedure handles Save node well.

The `MockQuantBackend` calls into `Interpreter::isOpSupported()`, so I think it makes more sense to add `SaveNode` there to return true, no?

Though I guess this is kind of moot as https://github.com/pytorch/glow/pull/2235 redesigns `Interpreter::isOpSupported()` anyway. Note that once that lands `Interpreter::isOpSupported()` will return true for `SaveNode`. If you want to leave it this way it's fine and I'll remove it in https://github.com/pytorch/glow/pull/2235 later! 😃

Reply:
Having check here will make sure that no matter what backend we use, SaveNode is not quantized, which what we want to check.

Yeah, I saw that PR and do not want to mess with it :)

Reply:
Well, the `MockQuantBackend` has hardcoded its "actual backend" as the Interpreter. But whatever it's NBD!